packages/cli/src/commands/integration/command.ts

```diff
@@ -91,6 +92,77 @@ export const listSubcommand = {
   ],
 } as const;
 
+export const removeSubcommand = {
+  name: 'remove',
+  description:
+    'Unlink and delete resources, and uninstall marketplace integrations',
+  arguments: [
+    {
+      name: 'resource/integration',
+      required: true,
+    },
+    {
+      name: 'project',
+      required: false,
+    },
+  ],
+  options: [
+    {
+      name: 'delete',
+      description: 'limits the resources listed to a designated integration',
```

Comment:
Is this description correct? I'm not sure why we mention listing with a `vc integration rm --delete`.

Reply:
Miss on copy-paste cleanup.